api/user/user.service.js

```diff
@@ -1,72 +1,43 @@
-import moment from 'moment';
-
-const users = [];
+import mongodbConnection from './../database/mongodb';
```

Comment:
Done in https://github.com/MakingSense/mern-seed/pull/18/commits/df45a349c21aad7b29d5e9db43056fcf17c6e6e4.